### ISSUE TITLE
UI doesn't keep selected preset after reopen it (Reaper 6.36 - x64)

### ISSUE THREAD

**Issue by linkoboy** [TheWaveWarden/odin2#397]
**Describe the bug**
UI doesn't keep selected preset after reopen it (Reaper 6.36 - x64)

**To Reproduce**
1. Load synth plugin
2. Select a preset
3. Close the UI
4. Reopen the UI
5. See the error
6. ![1](https://user-images.githubusercontent.com/40902122/132989647-f7ebdbad-e86a-42c2-ab90-539b0b9c5541.png)
![2](https://user-images.githubusercontent.com/40902122/132989649-30d726a4-c38e-4c18-b5a5-5f9556a081fe.png)

**Expected behavior**
I would like the last selected preset keeps selected in th UI.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: W10 64 bits
 - Reaper 6.36 - x64
 - Odin Version [e.g. 2.3.1]
 - Plugin Format [VST3]

**Comment by LiftPizzas** [comment]
It also happens if you switch from Odin in the FX window to another and then back. It's very frustrating trying to figure out which preset I had used.

**Comment by oiwn** [comment]
Hm it's same issue as for Dragonfly Reverbs and as far as i know it's only with vst3.

**Comment by linkoboy** [comment]
I feel like Odin2 has only a VST3 installer, the issue remains on version 2.3.4 :-(
Is there a vst2 installation available ?

**Comment by oiwn** [comment]
> I feel like Odin2 has only a VST3 installer, the issue remains on version 2.3.4 :-(
> Is there a vst2 installation available ?

I installed nightly build for OSX arm64 and don't see this issue.

https://github.com/TheWaveWarden/odin2/releases/tag/NightlyDevel you can try.

**Comment by TheWaveWarden** [comment]
@oiwn @linkoboy yes Odin 2 is VST3 only (because Steinberg ended licensing). You can build the project yourself in VST2 if you roll back to 2.2.X and follow the installation instructions in the repository. It won't make the preset appear though. This feature is not implemented currently its not about VST2/3.

**Comment by linkoboy** [comment]
> @oiwn @linkoboy yes Odin 2 is VST3 only (because Steinberg ended licensing). You can build the project yourself in VST2 if you roll back to 2.2.X and follow the installation instructions in the repository. It won't make the preset appear though. This feature is not implemented currently its not about VST2/3.

Thank for the details, I can't wait to see this feature for windows in the next releases :-)
